client/my-sites/purchases/payment-methods/index.tsx

```diff
@@ -99,8 +100,25 @@ export function AddNewPaymentMethod( { siteSlug }: { siteSlug: string } ): JSX.E
 	return (
 		<Main className="purchases is-wide-layout">
 			<MySitesSidebarNavigation />
-			<PageViewTracker path="/purchases/add-credit-card" title="Add Credit Card" />
-			<DocumentHead title={ translate( 'Add Credit Card' ) } />
+			<PageViewTracker
+				path={
+					config.isEnabled( 'purchases/new-payment-methods' )
+						? '/purchases/add-payment-method'
+						: '/purchases/add-credit-card'
```

Comment:
Correct me if I'm wrong, but `/purchases/add-credit-card` no longer exists, right? In this PR the route has been replaced with `/purchases/add-payment-method` (but maybe that wasn't intentional).

Reply:
Removing the old route was the mistake. I'll keep this in place and reinstate the old route.

Reply:
This should all be resolved with the latest commit.